e2etests/web/regular_integration_tests/README.md

```diff
@@ -1,22 +1,39 @@
-```
-This directory is for Flutter Web engine integration tests that does not
-need a specific configuration. If an e2e test needs specialized app
-configuration (e.g. PWA vs non-PWA packaging), please create another
-directory under e2etests/web. Otherwise tests such as text_editing, history,
-scrolling, pointer events... should all go under this package.
+# Flutter for Web Engine Integration testing
+
+This directory is for Flutter Web engine integration tests that does not need a specific configuration. If an integration test needs specialized app configuration (e.g. PWA vs non-PWA packaging), please create another directory under e2etests/web. Otherwise tests such as text_editing, history, scrolling, pointer events... should all go under this package.
+
+Tests can be run on both 'debug', 'release' and 'profile' modes. However 'release' mode will shorten the error. Use 'debug' mode for trouble-shooting purposes where you can also see the full stack trace.
 
-Tests can be run on both 'release' and 'profile' modes. However 'release' mode
-will shorten the error. Use 'profile' mode for trouble-shooting purposes where
-you can also see the full stack trace.
+## To run the application under test for troubleshooting purposes
 
-# To run the application under test for trouble shooting purposes.
+```
 flutter run -d web-server lib/text_editing_main.dart --local-engine=host_debug_unopt
+```
 
-# To run the Text Editing test and use the developer tools in the browser.
-flutter run --target=test_driver/text_editing_e2e.dart -d web-server --web-port=8080 --profile --local-engine=host_debug_unopt
+## To run the Text Editing test and use the developer tools in the browser
 
-# To test the Text Editing test with driver you either of the following:
-flutter drive -v --target=test_driver/text_editing_e2e.dart -d web-server --profile --browser-name=chrome --local-engine=host_debug_unopt
+```
+flutter run --target=test_driver/text_editing_integration.dart -d web-server --web-port=8080 --profile --local-engine=host_debug_unopt
```

Comment:
Does this one need `--target=`? (If so, why does the first example not use `--target`?)

Reply:
done.